### PR TITLE
School and nursery rated good

### PR DESCRIPTION
**Pull request by bailed22**
Transformation carried out as follows
1. datasets.json
2. editions
3. versions

### CODE REVIEW

**Review by GregoryJPavier — DISMISSED** [review]

File: src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    "@type": "hydra:Collection",
+    "title": "Schools and nursery schools rated good or outstanding",
+    "editions": [
+      {
+        "@id": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated/editions/2023-05",
+        "@type": "dcat:Dataset",
+        "in_series": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated",
+        "identifier": "2023-05",
+        "title": "Schools and nursery schools rated good or outstanding",
+        "summary": "Percentage of state-funded provided for schools that are rated",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json:13]: Maybe change this to 'Percentage of state-funded schools that are rated good or outstanding'

File: src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        "description": "This dataset shows the percentage of state-funded Nursery schools, Primary schools, Secondary schools, Alternative Provision (AP) schools and Special schools that are rated.",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json:14]: add 'good or outstanding' to the end of this description, give it a little more detail.

Comment (bailed22) [reply src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json:14]: okay

File: src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json (hunk 1 above)
Comment (bailed22) [reply src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json:13]: noted

File: src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            {
+              "name": "geography",
+              "datatype": "string",
+              "titles": "Geography Type",
+              "description": "The geography type of the area."
+            },
+            {
+              "name": "observation",
+              "datatype": "decimal",
+              "titles": "Percentage of state-funded schools",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json:79]: Maybe add 'good or outstanding' to the end of this too.

File: src/store/metadata/stub/content/editions/versions/schools-and-nursery-schools-rated_2023-05.json [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        {
+            "@type": [
+                "dcat:Distribution",
+                "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated/editions/2023-05/versions/1",
+            "identifier": "1",
+            "issued": "2023-05-16T13:36:00Z",
+            "title": "Schools and nursery schools rated good or outstanding",
+            "summary": "Percentage of state-funded provided for schools that are rated",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/versions/schools-and-nursery-schools-rated_2023-05.json:16]: Change this to 'Percentage of state-funded schools that are rated good or outstanding'

File: src/store/metadata/stub/content/editions/versions/schools-and-nursery-schools-rated_2023-05.json [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+            "description": "This dataset shows the percentage of state-funded Nursery schools, Primary schools, Secondary schools, Alternative Provision (AP) schools and Special schools that are rated.",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/versions/schools-and-nursery-schools-rated_2023-05.json:17]: Add 'good or outstanding' to the end of this description, give it a little more detail.

File: src/store/metadata/stub/content/editions/versions/schools-and-nursery-schools-rated_2023-05.json [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                {
+                  "name": "geography",
+                  "datatype": "string",
+                  "titles": "Geography Type",
+                  "description": "The geography type of the area."
+                },
+                {
+                  "name": "observation",
+                  "datatype": "decimal",
+                  "titles": "Percentage of state-funded schools",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/versions/schools-and-nursery-schools-rated_2023-05.json:49]: Maybe add 'good or outstanding' to the end of this too.

File: src/store/metadata/stub/content/datasets.json [hunk 7]
```diff
@@ -276,8 +276,53 @@
           "modified": "2023-11-02T14:41:11Z"
         }
       ]
+    },
+    {
+      "@id": "https://staging.idpd.uk/datasets/schools-and-nursery-schools-rated",
+      "@type": "dcat:DatasetSeries",
+      "identifier": "schools-and-nursery-schools-rated",
+      "title": "Schools and nursery schools rated good or outstanding",
+      "summary": "Percentage of state-funded provided for schools that are rated",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/datasets.json:285]: Change this to 'Percentage of state-funded schools that are rated good or outstanding'

File: src/store/metadata/stub/content/datasets.json [hunk 8]
```diff
@@ -276,8 +276,53 @@
...
+      "description": "This dataset shows the percentage of state-funded Nursery schools, Primary schools, Secondary schools, Alternative Provision (AP) schools and Special schools that are rated.",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/datasets.json:286]: Add 'good or outstanding' to the end of this description, give it a little more detail.

Comment (bailed22) [reply src/store/metadata/stub/content/datasets.json:286]: sorted and corrected

File: src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json (hunk 3 above)
Comment (bailed22) [reply src/store/metadata/stub/content/editions/schools-and-nursery-schools-rated_2023-05.json:79]: completed

File: src/store/metadata/stub/content/editions/versions/schools-and-nursery-schools-rated_2023-05.json (hunk 4 above)
Comment (bailed22) [reply src/store/metadata/stub/content/editions/versions/schools-and-nursery-schools-rated_2023-05.json:16]: completed

**Review by GregoryJPavier — COMMENTED** [review]
Nice, all looks good!

**Review by GregoryJPavier — APPROVED** [review]

**Review by GregoryJPavier — APPROVED** [review]

**Review by GregoryJPavier — APPROVED** [review]

**Pull request merged**